components/VM_Arm/src/main.c

```diff
@@ -927,6 +941,28 @@ static int load_vm(vm_t *vm, const char *kernel_name, const char *dtb_name, cons
     return 0;
 }
 
+static unsigned long parse_camkes_vm_entry_addr(void)
+{
+    unsigned long addr = strtoul(vm_address_config.kernel_entry_addr, NULL, 0);
+    unsigned long base = strtoul(vm_address_config.ram_base, NULL, 0);
+
+    /* Calculate an entry address if not explicitly set */
+    if (ENTRY_ADDR_NOT_SET == addr) {
+        if (config_set(CONFIG_PLAT_TX1) || config_set(CONFIG_PLAT_TX2) ||
+            config_set(CONFIG_PLAT_QEMU_ARM_VIRT) || config_set(CONFIG_PLAT_ODROIDC2) ||
+            config_set(CONFIG_PLAT_ZYNQMP)) {
+            /* This is likely an aarch64/aarch32 linux difference */
```

Comment:
We think so too, so how about using `CONFIG_ARCH_AARCH64` instead?

Reply:
So, let's take the chance and drop it - unless anybody (SIG-VMM?) has a strong interest to keep this?

Reply:
I'll add a commit dropping this for CONFIG_ARCH_AARCH64